microsoft-edge/devtools-guide-chromium/resources/index.md

```diff
@@ -22,89 +22,102 @@ ms.date: 05/04/2021
 *  HTML files.
 *  Image files.
 
 You can view the resources that make up a web page from within several tools, or panels.
 
-This guide assumes that you are familiar with the basics of [web development](https://developer.mozilla.org/docs/Learn) and [Microsoft Edge DevTools](../../devtools-guide-chromium/overview.md).
-
+This guide assumes that you are familiar with the basics of web development described in [Learn web development](https://developer.mozilla.org/docs/Learn) and [Microsoft Edge DevTools](../../devtools-guide-chromium/overview.md).
 
+To open DevTools, right-click the webpage, and then select **Inspect**. Or press `Ctrl`+ `Shift`+ `I` (Windows, Linux) or `Command`+ `Option`+ `I` (macOS).
 <!-- ====================================================================== -->
 ## Open resources from the Command Menu
 
-When you know the name of the resource that you want to inspect, the **Command Menu** provides a fast way of opening the resource.
+When you know the name of the resource that you want to inspect, the **Command Menu** in DevTools provides a fast way of finding and opening the resource.
+
+1. Press `Ctrl`+ `P` (Windows, Linux) or `Command`+`P` (macOS). You can also click the (![Customize icon.](../media/customize-devtools-icon-light-theme.png)) menu button and then select **Open file**. 
 
-1. Press `Ctrl`+`P` (Windows, Linux) or `Command`+`P` (macOS).  The **Open File** dialog opens.
+   The **Open file** dialog:
 
    ![The Open File dialog.](../media/resources-command-menu-empty.msft.png)
 
-1. Select the file from the dropdown list, or start typing the filename and press `Enter` once the correct file is highlighted in the autocomplete box.
+1. Start typing the filename and press `Enter` when the correct file is highlighted in the autocomplete box or select the file from the dropdown list.
+
+   Type filename:
 
    ![Type a filename in the Open File dialog.](../media/resources-command-menu-file-search.msft.png)
 
 
 <!-- ====================================================================== -->
 ## Open resources in the Network tool
 
-See [Inspect the details of a resource](../network/index.md#inspect-the-details-of-the-resource) in _Inspect network activity_.
+See [Inspect the details of the resource](../network/index.md#inspect-the-details-of-the-resource) in _Inspect network activity_ to learn how to get the details for a specific resource.
+
 <!--why?-->
 
-<!-- need lead-in -->
+Show network resource details:
 
 ![Inspect a resource in the Network tool.](../media/resources-network-response.msft.png)
 
-### Reveal resources in the Network tool from other panels
+#### Reveal resources in the Network tool from other panels
 
 The next section, [Browse resources in the Network panel](#browse-resources-in-the-network-panel), shows how to view resources from various parts of the DevTools UI.  To inspect a resource in the **Network** tool, right-click the resource and select **Reveal in Network panel**.
```

Comment:
Sentence 1 in this section currently reads "The next section,"
Sentence 1 in a section should describe/elaborate the present section heading, not a different section.  Add a Sentence 1 that clarifies the present heading, "Reveal resources in the Network tool from other panels".  What do we mean by "Reveal resources in the Network tool from other panels"?

Sentence 1 draft:
In several tools, such as X,Y,Z, you can right-click a page resource, such as a .js or .css file, to switch to the Network tool and list the selected resource within the Network tool.

Reply:
ok to close thread.  The Dec 9 review pass covers this comment.